src/DynamoCoreWpf/ViewModels/Core/NodeViewModel.cs

```diff
@@ -162,8 +162,11 @@ public bool IsSetAsInput
             {
                 if (nodeLogic.IsSetAsInput != value)
                 {
-                    nodeLogic.IsSetAsInput = value;
-                    RaisePropertyChanged("IsSetAsInput");
+                    DynamoViewModel.ExecuteCommand(
+               new DynamoModel.UpdateModelValueCommand(
```

Comment:
Can you check formatting here? If this was Python the statement would be outside the if :)